packages/TelegramClient-Tests.package/TCTChatsTests.class/instance/testPositionInsert.st

```diff
@@ -0,0 +1,9 @@
+testing
+testPositionInsert
+
+	self chats add: (TCCChat new
+		id: 0;
+		position: 0;
+		positionKnown: true).
+
+	self assert: self chats first id equals: 0.
```

Comment:
I think we want the assert to have the constant as the first argument.

Reply:
Resolved